### ISSUE TITLE
Binaries/Linux/unrealcv.ini doesn't exist

### ISSUE THREAD

**Issue by HenghuiB** [unrealcv/unrealcv#254]
- Operating System: Linux
- UE4 Version: 5.2
- UnrealCV Version:
- Client (python2, 3 or matlab):
- Problem Description:
In Binaries/Linux/unrealcv.ini doesn't exist, how could I change resolution and FOV?

**Comment by zfw1226** [comment]
`unrealcv.ini` will be created after you launch the binary. You can check the directory by running `vget /unrealcv/status` in the console.

**Comment by HenghuiB** [comment]
> `unrealcv.ini` will be created after you launch the binary. You can check the directory by running `vget /unrealcv/status` in the console.

Yeah, I use vget /unrealcv/status to get the path, but it doesn't exist

**Comment by wukui-muc** [comment]
On macOS, the inability to create this file is usually due to system permissions. A simple solution is to place the file in the appropriate directory. I have uploaded a `unrealcv.ini` file here. This method also applies to Linux.

[unrealcv.zip](https://github.com/user-attachments/files/19248713/unrealcv.zip)
